.env.example

```
@@ -0,0 +1,7 @@
+DATABASE_URL="postgresql://postgres:password@localhost:5432/db-name"
+BETTER_AUTH_SECRET="secret"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)